snapcraft/parts/grammar.py

```diff
@@ -68,7 +68,7 @@ def process_parts(
     processor = GrammarProcessor(
         arch=arch,
         target_arch=target_arch,
-        checker=lambda x: x == x,  # pylint: disable=comparison-with-itself
+        checker=lambda x: x == x,  # pylint: disable=comparison-with-itself  # noqa PLR0124
```

Comment:
Looks like black is unhappy with this line

Reply:
"Fixing one linter broke another linter"

A software engineering play in 2 acts

Reply:
OK, linter collisions fixed I think.